### PR TITLE
feat: Add Appsignal deploy marker

### PR DESCRIPTION
**Pull request by kimadactyl**
This just lets Appsignal know that there's a new deployment so we can see if things are fixed or not as a result of updates

### CODE REVIEW

**Review by katjam — APPROVED** [review]
Great idea.

File: config/appsignal.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# https://docs.appsignal.com/application/markers/deploy-markers.html
+
+production:
+  revision: <%= ENV['GIT_REV'] %>
```
Comment (katjam) [inline config/appsignal.yml:4]: I assume this is a Git env var set somewhere to track the latest tag / release?

**Comment by kimadactyl** [comment]
Oh no. An unrelated Capybara test has stopped working. I'm having a poke and it seems to have nothing to do with this. really weird.

**Comment by katjam** [comment]
> Oh no. An unrelated Capybara test has stopped working. I'm having a poke and it seems to have nothing to do with this. really weird.

Ugh.

File: test/system/application_system_test_case.rb [hunk 2]
```diff
@@ -9,4 +9,11 @@ class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
 
   Capybara.server = :puma, { Silent: true }
   Selenium::WebDriver.logger.ignore(:browser_options)
+
+  # TODO: Remove this hack
```
Comment (kimadactyl) [inline test/system/application_system_test_case.rb:13]: @katjam - not the prettiest but i think if we don't get this passing we can't deploy. i've made a tech debt ticket.

Comment (katjam) [reply test/system/application_system_test_case.rb:13]: Seems like a reasonable hack though.

**Pull request merged**